src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -105,6 +105,21 @@ protected override NamedTypeSymbol WithTupleDataCore(TupleExtraData newData)
 
         #region Syntax
 
+        public bool IsRecord()
```

Comment:
The reason I used a method rather than a property is this method doesn't just return a simple value, or compute a value and return the cached result.
Yes, I'm also not trying to bite the bullet of adding a general `IsRecord`

---
In reply to: [438385942](https://github.com/dotnet/roslyn/pull/45022#discussion_r438385942) [](ancestors = 438385942)